### DISCUSSION TITLE
roadmap

### DISCUSSION THREAD

**Discussion by dvlin-dev** [lin-stephanie/astro-antfustyle-theme#10] [General]
May I ask if there is a roadmap?

**Comment by dvlin-dev** [comment]
Thank you very much for your contribution. I plan to use this theme for my homepage; it's an awesome project.

**Reply by lin-stephanie** [reply]
Glad you like the theme and plan to use it. Thanks so much for your support!

**Comment by lin-stephanie** [comment]
A roadmap will soon be added to the Projects section on GitHub. Thanks again for following the project!

**Comment by lin-stephanie** [comment]
The Projects tab now has a Roadmap. Feel free to check it out, and share your thoughts!

**Reply by dvlin-dev** [reply]
Awesome, thanks for contributing, I love this new features haha!
